NuGet.config

```diff
@@ -8,5 +8,18 @@
     -->
     <add key=".Net Core Tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
   </packageSources>
+
+  <!-- Define mappings by adding package patterns beneath the target source. -->
+  <!-- *.Tools packages will be restored from ".Net Core Tools", everything else from nuget.org. -->
+  <packageSourceMapping>
+    <!-- key value for <packageSource> should match key values from <packageSources> element -->
+    <packageSource key="NuGet">
+      <package pattern="*" />
+    </packageSource>
+    <packageSource key=".Net Core Tools">
+      <package pattern="*.Tools" />
```

Comment:
The following error was thrown after central package management because `opentelemetry.net` is using more than one package source:

```sh
error NU1507: Warning As Error: There are 2 package sources defined in your configuration.
When using central package management, please map your package sources with package
source mapping (https://aka.ms/nuget-package-source-mapping) or specify a single package source.
The following sources are defined: NuGet, .Net Core Tools 
```

There are two solutions:

- To ignore the warning (as error) with `<NoWarn>$(NoWarn);NU1507</NoWarn>`. I don't think this is the best option;
- Enable [package source mapping](https://learn.microsoft.com/en-us/nuget/consume-packages/package-source-mapping#enabling-package-source-mapping).

As I'm choosing to follow the second approach, I need to know what packages (or package name pattern) should be restored by the `.Net Core Tools` source.

Reply:
FYI: @TimothyMothra

Reply:
I didn't know this repo had an additional PackageSource.
May need to open a new Issue to review this, this may not be needed anymore.

Looks like this was introduced in 2021 to use a 6.0.0-beta version of the ApiCompat tool. (https://github.com/open-telemetry/opentelemetry-dotnet/pull/2075/files)
However, this tool appears to be published publicly now and has newer versions. https://www.nuget.org/packages/Microsoft.DotNet.ApiCompat.Tool/